image/image_linux.go

```diff
@@ -163,8 +163,13 @@ func Prepare(opts *Options) error {
 		if model.Classic() {
 			return fmt.Errorf("cannot preseed the image for a classic model")
 		}
-		if model.Base() != "core20" {
-			return fmt.Errorf("cannot preseed the image for a model other than core20")
+
+		coreVersion, err := strutil.GetCoreVersion(model.Base())
```

Comment:
I am not sure `strutil` is a good fit for the GetCoreVersion() helper - it's mostly generic string function (VersionCompare is a notable exception) but this helper feels a bit too specific for strutil. I don't have a great suggestion, maybe just put it into `preseed.GetCoreVersion()` for now or we could create a `verutil` package with VersionCompare and this (but that also does not feel great).

Reply:
I'm inclined to put it into `preseed` for now but maybe @pedronis has a different sugestion?

Reply:
snap/naming has already WellKnownSnapID so maybe it can go there,  per golang conventions the Get in the name is not necessary, so naming.CoreVersion